pkgs/development/tools/agda-pkg/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, python3Packages, agda }:
+
+with python3Packages;
+
+buildPythonApplication rec {
+  pname = "agda-pkg";
+  version = "0.1.49";
+
+  disabled = !(pythonAtLeast "3.6");
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "07smmys9994gncsvd069iicyvj2k3yllwh77p6xr784bk11dnn8q";
+  };
+
+  doCheck = false;
```

at least ensure it executes.
```suggestion
  checkPhase = ''
    $out/bin/agda --help > /dev/null
```